kbkdf/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "kbkdf"
+version = "0.1.0"
+edition = "2018"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/RustCrypto/KDFs/"
+repository = "https://github.com/RustCrypto/KDFs/"
+description = "Key Derivation Using Pseudorandom Function (KBKDF)"
+keywords = ["crypto", "KBKDF", "KDF"]
+categories = ["cryptography", "no-std"]
+readme = "README.md"
+rust-version = "1.65"
+
+[dependencies]
+digest = { version = "0.11.0-pre.9", default-features = false, features = ["mac"] }
+
+# divrem can be dropped with MSRV >= 1.73
+divrem = "1.0.0"
```

Comment:
You might as well bump MSRV to 1.73. We will be bumping all crates to MSRV 1.85 for the 2024 edition for the next stable release.

Reply:
done
I bumped the MSRV to 1.81 like in other crates in the workspace.

Reply:
Great!